crates/starknet-devnet-core/src/starknet/dump.rs

```diff
@@ -133,6 +138,10 @@ impl Starknet {
         }
     }
 
+    pub fn dump_events_vec(&self) -> DevnetResult<Vec<DumpEvent>> {
```

Comment:
Let's not introduce Hungarian notation by mentioning the vector. Also, no need for this to return `Result` or `DevnetResult`. In fact, it could return a mere reference:
```rust
 pub fn read_dump_events(&self) -> &Vec<DumpEvent> {
```